lib/framework/http/controllers/logs.controller.js

```diff
@@ -97,56 +95,17 @@ const getAttachment = (request, response, next) => {
  *                        next middleware function.
  * @returns {undefined}
  */
-const index = async (request, response, next) => {
-    // TODO: Save final signature in commentary
-    // const index = async ({ persistence: { repositories: { LogRepository } } }, request, response, next) => {
-    // TODO: This call currently returns mock data, so this code is not required.
-    // Will have to be reverted later.
+const index = async ({ persistence: { repositories: { LogRepository } } }, request, response, next) => {
+    let logs;
+    try {
+        logs = await new GetAllLogsUseCase().execute();
```

Comment:
Missing dependency injection

Reply:
Could you elaborate?

Reply:
See the current code in master vs this change, you removed the setRepo call.

Reply:
The method wasn't working with this setRepo call. I had a look at the home controller and saw that the call worked without a setRepo call. So I removed it. I don't know how to implement this or why it is needed because the backend structure is completely new to me.

Reply:
@MauritsioRK like i said in an earlier comment change the InMemoryLogRepo

Reply:
@mvegter Its still not clear to me but is this what you meant?

Reply:
The `application/interfaces/**` define the interface of something, the implementation can be found in framework.

Reply:
Also, use the original method parameters

Reply:
@mvegter The persistence parameater does nothing currently and raises linting errors. Why should I keep it in?

Reply:
It actually did, but you added another import which overlaps.

Reply:
I renamed the import locally and still get linter errors.